definitions/procedures/backup/metadata.rb

```diff
@@ -65,8 +65,10 @@ def proxy_feature_list(spinner)
 
     def proxy_config(spinner)
       spinner.update('Collecting proxy configuration')
```

Comment:
Should this also be inside the `if` statement, just like all the other methods?

Reply:
I guess yeah